src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
I'm sure I've just done (or not done) something simple:
https://github.com/stephentoub/coreclr/commit/39dac79de49aaed46dd9b729c1b9afbd115d3781
I see the corefx.o file being produced.  Maybe I've just missed something in one of the make files?  Or maybe I'm misapplying the attribute? etc.